ci-operator/step-registry/origin/e2e/test/origin-e2e-test-ref.yaml

```diff
@@ -0,0 +1,12 @@
+ref:
+  as: origin-e2e-test
```

Comment:
We'll need to rename all these steps, this is not `origin`.  Probably this should be `openshift-test-e2e` and successive variants will be `openshift-test-e2e-serial` `openshift-test-e2e-disruptive`, etc.

The name of our platform is `openshift`, `okd` and `ocp` are variants that are expected to run and pass the same test suites.